pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -324,7 +324,7 @@ func newCertRotationController(
 			Name:      "service-network-serving-certkey",
 			AdditionalAnnotations: certrotation.AdditionalAnnotations{
 				JiraComponent:                    "kube-apiserver",
-				AutoRegenerateAfterOfflineExpiry: "https://github.com/openshift/cluster-kube-apiserver-operator/pull/1631,'operator conditions kube-apiserver'",
+				AutoRegenerateAfterOfflineExpiry: "https://github.com/openshift/cluster-kube-apiserver-operator/pull/1631,'[sig-apps] Deployment RollingUpdateDeployment should delete old pods and create new ones [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]'",
```

Comment:
I don't understand how this one tests the service network.  

1. test to external LB to create deployment
2. kcm uses internal LB to create replicaset and pod
3. scheduler uses internal LB to scheduler pod
4. kubelet uses internal LB to get pods and report available status
5. test uses external LB to confirm available replicas

which hop uses the service network?

Reply:
@vrutkovs, if you could provide an explanation, that would be very helpful. 

I also think that adding a comment with an explanation for each certificate/test pair would be greatly beneficial for future us and for easier reviewing.

Reply:
Ah, sorry, previously this was using
>[sig-apps] Deployment RollingUpdateDeployment should delete old pods and create new ones [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]

but deployment rollout doesn't use service network, instead its

> [sig-network] Services should serve a basic endpoint from pods [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]

now.

I'm not sure its worth commenting tests (we probably should have an easy to understand alias), but there will be an update for certificate descriptions: https://github.com/openshift/cluster-kube-apiserver-operator/pull/1763 - would it be sufficient? Do you want me to add comments on what these tests do in the code (and not touch the annotations)?